### PR TITLE
Discount features

### PR DESCRIPTION
**Pull request by UtsavSomaiya**

### CODE REVIEW

File: admin/settings.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            $fetch = $pdo->prepare("insert into discount(percentage) values('$discount')");
+            $result = $fetch->execute();
+            if (isset($result)) {
+                $_SESSION['msg'] = "Discount apply successfully";
+            }
+        }
+    } else {
+        $_SESSION['discountAlert'] = "Enter Only Numeric value";
+    }
+}
```
Comment (gauravmak) [inline admin/settings.php:28]: You must redirect after handling the POST request. How many times do we have to learn this?

File: admin/settings.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <div class="content-wrapper">
+        <div class="row">
+            <div class="col-md-6 grid-margin stretch-card">
+                <div class="card">
+                    <div class="card-body">
+                        <h4 class="card-title">Discount</h4>
+                        <form class="forms-sample" method="post" enctype="multipart/form-data">
+                            <div class="form-group">
+                                <label for="discount">Discount</label>
+                                <input id="discount" type="text" class="form-control" id="exampleInputUsername1" placeholder="Discount" name="discount" required>
```
Comment (gauravmak) [inline admin/settings.php:40]: exampleInputUsername1??

File: index.php [hunk 3]
```diff
@@ -103,6 +103,10 @@ function displayCart() {
             document.getElementById('div-main-' + id).appendChild(divMainSubThird);
 
             document.getElementById('subtotal').innerHTML = "$" + subTotal;
+
+            var discountPercentage = document.getElementById("discount-percentage").innerHTML.trim();
+            var discountPrice = subTotal * (discountPercentage / 100);
+            document.getElementById("discount-price").innerHTML = "- $" + discountPrice;
```
Comment (gauravmak) [inline index.php:109]: No need to decrease from total?

**Review by gauravmak — CHANGES_REQUESTED** [review]

File: admin/settings.php (hunk 1 above)
Comment (UtsavSomaiya) [reply admin/settings.php:28]: But sir aama redirect kya page pr kru

File: index.php (hunk 3 above)
Comment (UtsavSomaiya) [reply index.php:109]: Decrease In Main Total and this is new Module after this PR merged that we can create

File: admin/settings.php (hunk 1 above)
Comment (UtsavSomaiya) [reply admin/settings.php:28]: And aama type = number kri dav toh mare aa check j nhi krvu pde..

Comment (gauravmak) [reply admin/settings.php:28]: We have also discussed many times that you can redirect to the same page.

File: admin/settings.php (hunk 2 above)
Comment (gauravmak) [reply admin/settings.php:40]: Where is this change?

File: admin/settings.php (hunk 1 above)
Comment (UtsavSomaiya) [reply admin/settings.php:27]: Sir Redirect me kravyu toi refresh kru toh continue vadu aave che..

**Pull request merged**